tools/transform.cpp

```diff
@@ -564,7 +574,8 @@ static uint64_t get_access_size(const Type &ty) {
 static uint64_t get_access_size(const Instr &inst) {
   if (dynamic_cast<const Calloc*>(&inst) ||
       dynamic_cast<const Memcpy*>(&inst) ||
-      dynamic_cast<const Memset*>(&inst)) {
+      dynamic_cast<const Memset*>(&inst) ||
+      dynamic_cast<const Strlen*>(&inst)) {
```

Comment:
for now this is ok, but we need a subsequent patch to make it work with bits_byte > 8, since strlen is common.

Reply:
okay, understood